docs/reference/errors-and-warnings/NU3003.md

```diff
@@ -0,0 +1,44 @@
+---
+title: NuGet Warning NU3003
+description: NU3003 Warning code
+author: mishra14
+ms.author: anmishr
+manager: rrelyea
+ms.date: 8/16/2018
+ms.topic: reference
+ms.reviewer: anangaur
+f1_keywords:
+  - NU3003
+---
+
+# NuGet Warning NU3003
+
+## Scenario 1
+
+<pre>Package 'SamplePackage v1.0.0' from source 'http://source.com/index.json': The package is not signed. Unable to verify signature from an unsigned package.</pre>
```

Comment:
I don't follow this. 
Can't verify an unsigned package, and the solution is to file an issue? :)